gym/spaces/dict.py

```diff
@@ -19,23 +19,25 @@ class Dict(Space[TypingDict[str, Space]], Mapping):
 
     Example usage::
```

Comment:
Before this PR is merged I will check that this change works with the website:

```suggestion
    Example usage:
```